frontend/utils/filters.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
What's this file doing here? Remember from another PR it was already moved to `frontend/views/utils/filters.js`, what happened?

Reply:
Should remove it. Correct.

Reply:
Maybe `master` didn't get merged?

Reply:
I merged the previous branch. I manually removed this file because merging master branch won't delete this file, I think.